config/default_records/plate_purposes/014_bge.wip.yml

```diff
@@ -0,0 +1,23 @@
+# Plate purposes for BGE
+---
+BGE Cherrypick:
+  type: PlatePurpose::Input
+  target_type: Plate
+  stock_plate: true
+  default_state: passed
+  cherrypickable_target: true
+  size: 96
+BGE Lib XP2:
+  type: PlatePurpose::Input
+  target_type: Plate
+  stock_plate: true
```

Comment:
I don't think BGE Lib XP2 or BGE Lib PCR XP are stock_plate's. @KatyTaylor might know a bit more about these attributes.

Reply:
BGE Cherrypick should be an 'input' plate (as you have done) because it's the first one in the pipeline. I'm not sure whether it's necessary for Lib XP2 and Lib PCR XP or not. If it was me I'd start of without and add it back in later if necessary.

I don't think any of them need to be 'stock' plates. A plate needs to be a 'stock' if it's used in a sample manifest - none of these are. And I don't think they would be considered stock plates by the lab as they're not the ones that came directly from the customer.

Reply:
LB Cherrypick and PF Cherrypicked (that this plate is modelled off) are both set as [stock_plate: true](https://github.com/sanger/sequencescape/blob/develop/config/default_records/plate_purposes/005_limber_purposes.yml#L9).